packages/server/package.json

```diff
@@ -89,6 +89,7 @@
     "node": ">=12.0"
   },
   "dependencies": {
+    "@apollo/cache-control-types": "^1.0.1",
```

Comment:
Does this actually need to be a dependency in server? We only import types from it in runtime code. We use the functions in test code but that can still be a top-level dev dependency.

Reply:
Good question, but I think the answer is yes, as the exported type `GraphQLRequestContext` has an `overallCachePolicy: CachePolicy` field.

However, I think the point holds for plugin-response-cache, so I'll remove it from there.